### PR TITLE
Session timetable + managers + coordinators

### PR DESCRIPTION
**Pull request by OmeGak**

### CODE REVIEW

File: indico/modules/attachments/util.py [hunk 1]
```diff
@@ -104,8 +104,6 @@ def can_manage_attachments(obj, user):
         return True
     if isinstance(obj, db.m.Event) and obj.can_manage(user, 'submit'):
         return True
-    if isinstance(obj, db.m.Session) and obj.can_manage(user, 'coordinate'):
```
Comment (ThiefMaster) [inline indico/modules/attachments/util.py]: why? this is for adding attachments to a session!

File: indico/modules/events/timetable/blueprint.py [hunk 2]
```diff
@@ -36,22 +37,20 @@
 _bp.add_url_rule('/manage/timetable/', 'management', RHManageTimetable)
 _bp.add_url_rule('/manage/timetable/', 'timetable_rest', RHTimetableREST, methods=('POST',))
 _bp.add_url_rule('/manage/timetable/<int:timetable_entry_id>', 'timetable_rest', RHTimetableREST, methods=('PATCH',))
+_bp.add_url_rule('/manage/timetable/session/<int:session_id>', 'manage_session', RHManageSessionTimetable)
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/blueprint.py]: i'd add a trailing slash like we have for `/manage/timetable/` itself

File: indico/modules/events/timetable/templates/_timetable.html [hunk 3]
```diff
@@ -10,20 +11,36 @@
 
     <script>
         $(window).on('load', function() {
-            var timetableLayout = {{ timetable_layout|tojson }};
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/templates/_timetable.html]: `'use strict';`

File: indico/modules/attachments/util.py (hunk 1 above)
Comment (OmeGak) [inline indico/modules/attachments/util.py]: Why would the coordinator be able to add session-wide (manager-only actions) attachments?

Comment (ThiefMaster) [inline indico/modules/attachments/util.py]: I don't remember what exactly but there was a usecase for it so let's not remove it for now.

**Pull request closed without merging**